<launch>
    <arg name="name_space" default="iai_donbot/whole_body_controller/base"/>
    <rosparam command="load" ns="iai_donbot" file="$(find omni_pose_follower)/config/ros_control_sim.yaml" />
    <node pkg="omni_pose_follower" type="omni_pose_follower.py" name="omni_pose_follower" output="screen">
        <remap from="~/cmd_vel" to="/base_controller/command"/>
        <remap from="/joint_states" to="/iai_donbot/joint_states"/> 
        <remap from="~follow_joint_trajectory" to="/iai_donbot/whole_body_controller/base/follow_joint_trajectory"/>
        <remap from="~state" to="/iai_donbot/whole_body_controller/base/state"/>
        <param name="name_space" value="$(arg name_space)"/>
        <param name="vel_tolerance" value="0.01"/>
    </node>
</launch>